src/wallet.ts

```diff
@@ -590,14 +600,37 @@ async function esploraTxToTxInterface(
   );
 
   const txHex = await fetchTxHex(esploraTx.txid, explorerUrl);
-  const txOutputs = Transaction.fromHex(txHex).outs.map(
-    txOutputToOutputInterface
-  );
+  const transaction = Transaction.fromHex(txHex);
```

Comment:
The fee amount is already present in the EsploraTx (`fee: number` at L509), no need to decode from raw tx hex and loop all outs.

Also the asset is always LBTC, so need to exposing it.